StyleCop.Analyzers/StyleCop.Analyzers.CodeFixes/NamingRules/SA1308CodeFixProvider.cs

```diff
@@ -66,16 +60,24 @@ public override FixAllProvider GetFixAllProvider()
                     break;
                 }
 
-                if (!string.IsNullOrEmpty(token.ValueText))
+                if (numberOfCharsToRemove == 0)
+                {
+                    continue;
```

Comment:
@sharwell I think I added this to maintain previous behavior if the name was 1 char, or 2 chars and not one of the prefixes; without this check, we would no longer hit the `continue` below where we had before. However, if either of those were the case, then clearly the analyzer wouldn't have reported this in the first place. I'll remove this `if` block.